res/compile.py

```diff
@@ -0,0 +1,193 @@
+import os
+import subprocess
+import re
+import binascii
+import struct
+from multiprocessing.pool import ThreadPool
+
+# Set the path to the fxc compiler
+fxc = 'C:\\Program Files (x86)\\Microsoft DirectX SDK (June 2010)\\Utilities\\bin\\x86\\fxc.exe'
```

Comment:
@Xottab-DUTY do you suggest using raw string literals? ```"""text"""```? That's arguable. It won't make the string shorter.